bionic/protocols.py

```diff
@@ -197,6 +198,29 @@ def __repr__(self):
         return f"{self.__class__.__name__}(...)"
 
 
+class JsonProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values can be serialized using the
+    ``json`` library.
```

Comment:
I think the important thing here is that we are serializing to the JSON format -- not the particular library we're using.